tests/integration/mongodb/phpunit.xml.dist

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file does not represent the structure of directories described. Have a look at the CircleCI tests, you will see that they have warnings.